builtin_interfaces/Time stamp
uint16[] id_list

uint16[] port
string[] mode
uint16[] data

string MODE_ANALOG_IN           = "a_in";
string MODE_DIGITAL_OUT         = "d_out";
string MODE_DIGITAL_IN_PULLUP   = "d_in_pullup";
string MODE_DIGITAL_IN_PULLDOWN = "d_in_pulldown";